src/edu/asu/plp/tool/prototype/Main.java

```diff
@@ -413,45 +463,843 @@ private ProjectExplorerTree createProjectTree()
+		ASMFile activeFile = openProjects.getKey(selectedTab);
+		// TODO: check activeFile for null-value
+		return activeFile.getProject();
+	}
+	
+	private void onSimProjectClicked(MouseEvent event, HBox toolbar)
+	{
+		DropShadow ds = new DropShadow();
+		if (!simMode)
+		{
+			for (int x = 9; x <= 23; x++)
+			{
+				toolbar.getChildren().get(x).setEffect(null);
+				toolbar.getChildren().get(x).setDisable(false);
+			}
+			simMode = true;
+		}
+		else
+		{
+			for (int x = 9; x <= 23; x++)
+			{
+				toolbar.getChildren().get(x).setEffect(ds);
+				toolbar.getChildren().get(x).setDisable(true);
+			}
+			simMode = false;
+		}
+	}
+	
+	private void onlineManualWeb()
+	{
+		try
+		{
+			if (Desktop.isDesktopSupported())
+			{
+				Desktop.getDesktop().browse(new URI(
+						"https://code.google.com/p/progressive-learning-platform/wiki/UserManual"));
+			}
+		}
+		catch (Exception e)
+		{
+			e.printStackTrace();
+		}
+	}
+	
+	private void createNewProject()
+	{
+		Stage createProjectStage = new Stage();
+		Parent myPane = projectCreateMenu();
+		Scene scene = new Scene(myPane, 450, 350);
+		createProjectStage.setTitle("Create New PLP Project");
+		createProjectStage.setScene(scene);
+		createProjectStage.setResizable(false);
+		createProjectStage.show();
+		
+	}
+	
+	private Parent projectCreateMenu()
+	{
+		BorderPane border = new BorderPane();
+		border.setPadding(new Insets(20));
+		GridPane grid = new GridPane();
+		HBox buttons = new HBox(10);
+		grid.setHgap(10);
+		grid.setVgap(30);
+		grid.setPadding(new Insets(10, 10, 10, 10));
+		
+		Label projectName = new Label();
+		projectName.setText("Project Name: ");
+		projectName.setFont(Font.font("Arial", FontWeight.NORMAL, 16));
+		
+		TextField projTextField = new TextField();
+		projTextField.setText("Project Name");
+		projTextField.requestFocus();
+		projTextField.setPrefWidth(200);
+		
+		Label mainSourceFile = new Label();
+		mainSourceFile.setText("File Name: ");
+		mainSourceFile.setFont(Font.font("Arial", FontWeight.NORMAL, 16));
+		
+		TextField sourceFileField = new TextField();
+		projTextField.setPrefWidth(200);
+		
+		Label projectLocation = new Label();
+		projectLocation.setText("Location: ");
+		projectLocation.setFont(Font.font("Arial", FontWeight.NORMAL, 16));
+		
+		TextField projLocationField = new TextField();
+		projTextField.setPrefWidth(200);
+		
+		Button browseLocation = new Button();
+		browseLocation.setText("Browse");
+		browseLocation.setOnAction(new EventHandler<ActionEvent>() {
+			@Override
+			public void handle(ActionEvent e)
+			{
+				String chosenLocation = "";
+				FileChooser fileChooser = new FileChooser();
+				// fileChooser.setTitle("Choose Project Location");
+				DirectoryChooser directoryChooser = new DirectoryChooser();
+				//directoryChooser.getExtensionFilters().add(new
+				// FileChooser.ExtensionFilter("PLP files (*.plp)", "*.plp"));
+				// directoryChooser.setInitialDirectory(projTextField.getText());
```

Comment:
Before this goes to master, commented code needs to be removed, unless it's part of a comment to clarify existing code